adapter/memQuota/memQuota_test.go

```diff
@@ -335,6 +345,54 @@ func TestReaper(t *testing.T) {
 	}
 }
 
+func TestReaperTicker(t *testing.T) {
```

Comment:
I'm not a fan of the sleep, but I *am* a big fan of 100% coverage. I'm happy to entertain additional options ;-)